autoload/db_ui/table_helpers.vim

```diff
@@ -1,4 +1,4 @@
-let s:basic_constraint_query = "
+let s:basic_foreign_key_query = "
```

Comment:
Why did you rename this variable?

Reply:
Oh, I must have done this by mistake and didn't realise it. I will correct this change.

Reply:
Hi! Thanks for response. I see that this change is already merged into main branch in commit [65454e2](https://github.com/kristijanhusak/vim-dadbod-ui/commit/65454e2011c029922c2c972d63f8bf04282de033) from this [PR  #264](https://github.com/kristijanhusak/vim-dadbod-ui/pull/264/files)

I'm not sure why don't know in my PR :thinking: 

What can I do to proceed?

Reply:
Ok, I think I found it! My fork wasn't synced, it should be fine now. Sorry about that.